examples/community/stable_diffusion_tensorrt_txt2img.py

```diff
@@ -746,6 +748,7 @@ def to(self, torch_device: Optional[Union[str, torch.device]] = None, silence_dt
 
         return self
 
+    # Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline
```

Comment:
This `__encode_prompt` function cannot be copied from `StableDiffusionPipeline` because they are different. `__encode_prompt` and `__decode_latent` functions are `TensorRTStableDiffusionPipeline`'s own functions, try not to touch them.